jib-core/src/main/java/com/google/cloud/tools/jib/api/JavaContainerBuilder.java

```diff
@@ -264,15 +273,65 @@ public JavaContainerBuilder addDependencies(List<Path> dependencyFiles) throws I
   }
 
   /**
-   * Adds dependency JARs to the image. Duplicate JAR filenames are renamed with the filesize in
-   * order to avoid collisions.
+   * Convenience alternative to {@link #addDependencies(List)}.
```

Comment:
I think we should just copy-paste the javadoc from the original method. (Same on L302 and L330)

Reply:
Yeah I took it out because I didn't like the duplication, but I think you're right.